e2e/server/core-requirements.txt

```diff
@@ -0,0 +1,3 @@
+gunicorn==19.7.1
+honcho==1.0.1
+git+https://github.com/superdesk/superdesk-core.git@develop#egg=superdesk-core
```

Comment:
We should probably move this requirement to `setup.py`, so it's not required here nor in `server/requirements.txt`

Reply:
I actually wanted to have it separate so it can be cached by docker